weave-js/src/util/refs.ts

```diff
@@ -0,0 +1,28 @@
+const PROTOCOL = 'weave://';
+
+export const makeRefCall = (
```

Comment:
These two small utility functions aren't used in this PR but will be in upcoming feedback UI code.